api/db/migrations/20240910142209_create_script-execution-history_table.js

```diff
@@ -0,0 +1,18 @@
+const TABLE_NAME = 'script_exec_history';
```

Comment:
Proposition de nom pour la table :
```suggestion
const TABLE_NAME = 'script_executions';
```